### ISSUE TITLE
RFE: add speech-dispatcher plugin

### ISSUE THREAD

**Issue by nullr0ute** [MycroftAI/mimic#91]
speech-dispatcher is a high-level device independent layer for TTS and is the universal interface that is used by GNOME and KDE for desktop speech integration.

https://devel.freebsoft.org/speechd

It supports a plugin interface so I believe a mimic speechd plugin would be a good initial integration for Mycroft/minic integration for general desktops.

**Comment by forslund** [comment]
Thanks for the suggestion, sounds like a good fit for mimic. Is this something you have experience with and would like to contribute?

**Comment by nullr0ute** [comment]
Sadly my level of experience is more at the distro maintenance level of speech-dispatcher. I'm the Fedora maintainer of it and have debugged and done small bits but no major feature development on it. There is a FLITE plugin so I'm not sure how diverged mimic is from that but it might be a useful starting point.

**Comment by forslund** [comment]
Yeah, the api hasn't diverged very much from flite yet so it will definitely be a good starting point. Thanks.
